### PR TITLE
[DO NOT GO LIVE] add JVM_HEAP_SIZE config doc

### PR DESCRIPTION
**Pull request by junejung**
# Description
A brief description of the changes.

Add new doc to explain how to configure JVM_HEAP_SIZE

# Reasons
Why did you make these changes?

as part of server 2.15 release, we are added a new change to  enable the operator to reset JVM_HEAP_SIZE

# Context
A link to a GitHub and/or JIRA issue (if applicable).
https://circleci.atlassian.net/browse/CIRCLE-12870

### CODE REVIEW

File: jekyll/_cci2/jvm-heap-size-configuration.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```sh
+sudo docker exec -it test-result lein repl :connect 2719
+```
+
+verify JVM_HEAP_SIZE has reset.
+```clojure
+(System/getenv "JVM_HEAP_SIZE") ;; should return what you have set above
+```
+```clojure
+(-> (java.lang.Runtime/getRuntime) (.totalMemoryMemory)) ;; return value should match with JVM_HEAP_SIZE
```
Comment (polymeris) [inline jekyll/_cci2/jvm-heap-size-configuration.md]: `.totalMemoryMemory` should probably just be `.totalMemory`

File: jekyll/_cci2/jvm-heap-size-configuration.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+To customize JVM_HEAP_SIZE value, you will need to create customizations file in your services box
+1. Create customizations files for frontend or test-result
+```sh
+/etc/circleconfig/frontend/customizations
+/etc/circleconfig/test-results/customizations
+```
+
+2. In the file, add the line below to export desire JVM_HEAP_SIZE in the file
+```sh
+export JVM_HEAP_SIZE=<desiredsize>
```
Comment (polymeris) [inline jekyll/_cci2/jvm-heap-size-configuration.md]: Maybe specify what unit this size is in. Possibly add an example.

File: jekyll/_cci2/jvm-heap-size-configuration.md (hunk 1 above)
Comment (polymeris) [reply jekyll/_cci2/jvm-heap-size-configuration.md]: https://docs.oracle.com/javase/7/docs/api/java/lang/Runtime.html#totalMemory()

File: jekyll/_cci2/jvm-heap-size-configuration.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+To customize JVM_HEAP_SIZE value, you will need to create customizations file in your services box
+1. Create customizations files for frontend or test-result
+```sh
+/etc/circleconfig/frontend/customizations
+/etc/circleconfig/test-results/customizations
+```
+
+2. In the file, add the line below to export desire JVM_HEAP_SIZE(e.g. 2g, 512m) in the file
+```sh
+export JVM_HEAP_SIZE=<desiredsize>
```
Comment (MikeyYeahYeah) [inline jekyll/_cci2/jvm-heap-size-configuration.md]: I always like to include an example when possible.  In this case maybe something like:
`export JVM_HEAP_SIZE=2g`

**Pull request merged**